audformat/core/database.py

```diff
@@ -935,6 +945,11 @@ def load(
         Returns:
             database object
 
+        Raises:
+            FileNotFoundError: if a file associated with an attachment
+                cannot be found
+                or is not a file
```

Comment:
```suggestion
                or is not a file or folder
```

Reply:
Or actually I think we can remove "or is not a file" now that we support folders?

Reply:
Which I also wrote for `save()`.